botorch/utils/sample_pareto.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+Some methods for sampling the Pareto optimal points. This relies on the pymoo
```

Comment:
It seems like this should live in botorch/utils/multi_objective/sample_pareto.py